src/ops/array_ops_test.ts

```diff
@@ -511,6 +511,16 @@ describeWithFlags('onesLike', ALL_ENVS, () => {
     expectArraysEqual(b, [1, 1, 1]);
   });
 
+  it('1D complex dtype', () => {
+    const real = tf.tensor1d([1, 2, 3], 'float32');
+    const imag = tf.tensor1d([1, 2, 3], 'float32');
+    const a = tf.complex(real, imag);
+    const b = tf.onesLike(a);
+    expect(b.dtype).toBe('complex64');
+    expect(b.shape).toEqual([3]);
+    expectArraysEqual(b, [1, 0, 1, 0, 1, 0]);
```

Comment:
shouldn't these all be ones?

Reply:
`ones_like` of TensorFlow core generates a complex number with real 1 and imaginary 0. So I think it's consistent with TensorFlow core implementation. 

```
>>> tf.ones_like([1,2,3], dtype='complex64')
<tf.Tensor: id=18, shape=(3,), dtype=complex64, numpy=array([1.+0.j, 1.+0.j, 1.+0.j], dtype=complex64)>
```